kola/tests/ostree/basic.go

```diff
@@ -277,16 +293,23 @@ func ostreeRemoteTest(c cluster.TestCluster) {
 	c.Run("delete", func(c cluster.TestCluster) {
 		preRemotesOut := c.MustSSH(m, "ostree remote list")
 		preNumRemotes := len(strings.Split(string(preRemotesOut), "\n"))
+
 		if preNumRemotes < 1 {
 			c.Fatalf(`No remotes configured on host: %q`, string(preRemotesOut))
 		}
 
 		c.MustSSH(m, ("sudo ostree remote delete " + remoteName))
 
-		delRemoteListOut := c.MustSSH(m, "ostree remote list")
-		delNumRemotes := len(strings.Split(string(delRemoteListOut), "\n"))
+		delRemoteListOut := string(c.MustSSH(m, "ostree remote list"))
```

Comment:
Why not use `getOstreeRemotes` for initializing this variable & `preNumRemotes`?

Reply:
Only because of the failure message printing out the raw output. I can update `getOstreeRemotes` to return the output and the number and re-use it. Will update shortly.

Reply:
Updated